static/css/styles.css

```diff
@@ -172,3 +172,17 @@
     display: block;
     margin-bottom: 5px;
 }
+
+/* CE-201-PV */
+.CE-201-PV .group {
+    display: table;
+}
+
```

Comment:
Move this to base.html. externalised static files have an issue #217

Reply:
Done